### DISCUSSION TITLE
Releases Documentation

### DISCUSSION THREAD

**Discussion by MikeKatsoulakis** [jamesmh/coravel#412] [Q&A] [answered]
Hey there, is there a page or a wiki with the changes (either breaking or not) for each version?
I am using Coravel 5.0.4 and I saw that 6.0 got released and I was wondering what are the changes.

Thank you!

**Comment by jamesmh** [comment, marked as answer]
There's no explicit list, but here are the changes for Coravel specifically (excluding mailer + caching):

- https://docs.coravel.net/Scheduler/#logging-tick-catch-up
- Change to `LogScheduledTaskProgress()` method signature
